DANS-KNAW/easy-manage-deposit
# Easy 1822 easy-manage-deposit-app should not exit when deposit.properties are not found while deleting

Fixes EASY-1822

#### When applied it will
* prevent the program from exiting when a NotReadableException occurs while deleting deposits

#### Where should the reviewer @DANS-KNAW/easy start?
#### How should this be manually tested?
Delete deposits without properties or user rights and see if the program continues 
#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-manage-deposit                      | [PR#35](https://github.com/DANS-KNAW/easy-manage-deposit/pull/35)     | ..
